src/shop/Shop.tsx

```diff
@@ -17,20 +19,36 @@ const SECTIONS: TRewardSectionProps[] = [
   { filter: 'populair', title: 'SHOP.SECTIONS.POPULAR' },
   { filter: 'sport', horizontal: true, title: 'SHOP.SECTIONS.SPORTIVE' },
   { category: 'Goede doel', title: 'SHOP.SECTIONS.CHARITY' },
-]
+];
 
-const Shop = () => {
+type TProps = {
+  navigation: TMainNavigationProp<'Shop'>;
+};
+
+const Shop = ({ navigation }: TProps) => {
   const { t } = useTranslation();
   const { data: user } = useGetMe();
 
   return (
-    <SafeAreaView edges={['left', 'right']} isScrollable>
-      <WelcomeGiftsBanner />
-      <CategoryFilters />
-
-      {SECTIONS.map(({ filter, category, title, horizontal }) => (
-        <RewardsSection category={category} filter={filter} horizontal={horizontal} key={title} title={t(title, { city: user?.address?.city || t('SHOP.SECTIONS.CITY_FALLBACK') })} />
-      ))}
+    <SafeAreaView edges={['left', 'right']} isScrollable stickyHeaderIndices={[]}>
+      <EnlargedHeader height={30} />
+      <Styled.SearchContainer>
+        <Styled.SearchInput onFocus={() => navigation.push('Search')} placeholder={t('SHOP.SEARCH.PLACEHOLDER')} value="" />
```

Comment:
Het lijkt mij "gevaarlijk" om hier echt een input te gebruiken... Ik vrees dat ge ooit eens in een situatie terecht gaat komen waarbij ge hier toch in focussed raakt en dus uw keyboard open springt en ge kunt typen maar dat dus niets gaat doen